community/c++-algo/include/GameLib/src/enums.h

```diff
@@ -44,5 +44,10 @@ namespace terminal {
         CRASH,
     };
 
+    enum MOVE_DIRECTION {

```

Comment:
I would move this into the navigation.h file, since it is not used anywhere outside of navigation.